dispatch/static/manager/src/js/components/ArticleEditor/index.js

```diff
@@ -22,6 +23,8 @@ class ArticleEditorComponent extends React.Component {
     super(props)
 
     this.toggleStyle = this.toggleStyle.bind(this)
+
+    this.state = { isSaved: true }
```

Comment:
sure. I suppose that keeps it in check with the existing architecture.
I can make the change on Monday or you can do it.